docs/docs/configuration/object_detectors.md

```diff
@@ -682,6 +682,29 @@ model:
 
 Note that the labelmap uses a subset of the complete COCO label set that has only 80 objects.
 
+#### YOLOx
+
+[YOLOx](https://github.com/Megvii-BaseDetection/YOLOX) models are supported, but not included by default. See [the models section](#downloading-yolo-models) for more information on downloading the YOLOx model for use in Frigate.
+
+After placing the downloaded onnx model in your config folder, you can use the following configuration:
+
+```yaml
+detectors:
+  onnx:
+    type: onnx
+
+model:
+  model_type: yolox
+  width: 416 # <--- should match the imgsize set during model export
+  height: 416 # <--- should match the imgsize set during model export
+  input_tensor: nchw_denorm
+  input_dtype: float
```

Comment:
This seems to need to be set to `float_denorm` as this PR introduced support for that? 

I didn't get objects detected with just `float`, but it works with `float_denorm`!

Reply:
yes